### ISSUE TITLE
App crashes on start

### ISSUE THREAD

**Issue by pixorama** [telegramdesktop/tdesktop#26566]
### Steps to reproduce

Just open Telegram

### Expected behaviour

Telegram works

### Actual behaviour

Telegram crashes a few seconds after launch

### Operating system

Windows 10 Enterprise Version 21H2 Build 19044.3208

### Version of Telegram Desktop

x64 4.8.7 (regular and portable)

### Installation source

Static binary from official website

### Crash ID

Assertion Failed! "value.bytes.size() <= _settings.maxDataSize" storage_cache_database_object.cpp:807

### Logs

```shell
[2023.07.24 12:32:26] Launched version: 4008007, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2023.07.24 12:32:26] Executable dir: C:/Program Files/Telegram/, name: Telegram.exe
[2023.07.24 12:32:26] Initial working dir: C:/Program Files/Telegram/
[2023.07.24 12:32:26] Working dir: C:/Program Files/Telegram/
[2023.07.24 12:32:26] Command line: C:\Program Files\Telegram\Telegram.exe
[2023.07.24 12:32:26] Executable path before check: C:/Program Files/Telegram/Telegram.exe
[2023.07.24 12:32:26] Logs started
[2023.07.24 12:32:26] Connecting local socket to Global\0232e595ddf3b7bd4fdc1046e724d04e-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2023.07.24 12:32:26] This is the only instance of Telegram, starting server and app...
[2023.07.24 12:32:26] Moved logging from 'C:/Program Files/Telegram/log_start0.txt' to 'C:/Program Files/Telegram/log.txt'!
[2023.07.24 12:32:26] Global devicePixelRatio: 1
[2023.07.24 12:32:26] QT_DPI_ADJUSTMENT_POLICY: AdjustDpi
[2023.07.24 12:32:26] Primary screen DPI: 96, Base: 96.
[2023.07.24 12:32:26] Computed screen scale: 100
[2023.07.24 12:32:26] DevicePixelRatio: 1
[2023.07.24 12:32:26] ScreenScale: 100
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.07.24 12:32:26] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.24 12:32:26] Using DirectX compiler 'C:\Program Files\Telegram\modules\x64\d3d\d3dcompiler_47.dll'.
[2023.07.24 12:32:26] Checking...
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\.
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\..
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\desktop.ini
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\Far Manager 3 (x64).lnk
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\File Explorer.lnk
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\Google Chrome.lnk
[2023.07.24 12:32:26] Checking C:\Users\user\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\Tombstones
[2023.07.24 12:32:27] OpenAL Logging Level: (not set)
[2023.07.24 12:32:27] Audio Playback Devices: OpenAL Soft on Speakers (Realtek High Definition Audio);OpenAL Soft on Realtek Digital Output (Realtek High Definition Audio)
[2023.07.24 12:32:27] Audio Playback Default Device: OpenAL Soft on Speakers (Realtek High Definition Audio)
[2023.07.24 12:32:27] Audio Capture Devices: 
[2023.07.24 12:32:27] Audio Capture Default Device: 
[2023.07.24 12:32:27] Audio Error: No capture device found!
[2023.07.24 12:32:28] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2023.07.24 12:32:28] OpenGL Profile: OpenGLES.
[2023.07.24 12:32:28] OpenGL Renderer: ANGLE (AMD, AMD Radeon (TM) R9 390 Series Direct3D11 vs_5_0 ps_5_0, D3D11-27.20.1034.6)
[2023.07.24 12:32:28] OpenGL Vendor: Google Inc. (AMD)
[2023.07.24 12:32:28] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2023.07.24 12:32:28] OpenGL Extensions: GL_ANGLE_memory_size, GL_OES_texture_border_clamp, , GL_OES_rgb8_rgba8, GL_EXT_draw_buffers, GL_ANGLE_program_cache_control, GL_ANGLE_texture_compression_dxt5, GL_OES_vertex_array_object, GL_ANGLE_explicit_context_gles1, GL_ANGLE_provoking_vertex, GL_EXT_frag_depth, GL_EXT_texture_compression_rgtc, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_CHROMIUM_copy_texture, GL_OES_surfaceless_context, GL_OES_compressed_ETC2_sRGB8_texture, GL_OES_texture_npot, GL_ANGLE_framebuffer_multisample, GL_ANGLE_lossy_etc_decode, GL_ANGLE_explicit_context, GL_OES_texture_stencil8, GL_OES_EGL_image_external_essl3, GL_EXT_discard_framebuffer, GL_WEBGL_video_texture, GL_ANGLE_instanced_arrays, GL_EXT_draw_buffers_indexed, GL_KHR_debug, GL_ANGLE_depth_texture, GL_EXT_color_buffer_half_float, GL_EXT_disjoint_timer_query, GL_EXT_texture_norm16, GL_EXT_multisampled_render_to_texture, GL_EXT_occlusion_query_boolean, GL_ANGLE_multiview_multisample, GL_EXT_unpack_subimage, GL_OES_texture_half_float_linear, GL_ANGLE_texture_multisample, GL_CHROMIUM_lose_context, GL_OES_fbo_render_mipmap, GL_OES_texture_float_linear, GL_EXT_debug_label, GL_OES_compressed_EAC_R11_unsigned_texture, GL_NV_pack_subimage, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_EXT_blend_minmax, GL_EXT_float_blend, GL_OES_compressed_EAC_RG11_signed_texture, GL_ANGLE_copy_texture_3d, GL_ANGLE_get_serialized_context_string, GL_OES_compressed_EAC_R11_signed_texture, GL_OES_mapbuffer, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_OVR_multiview, GL_ANGLE_framebuffer_blit, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_draw_buffers_indexed, GL_OES_texture_half_float, GL_ANGLE_texture_usage, GL_NV_EGL_stream_consumer_external, GL_OES_EGL_image, GL_OES_compressed_ETC2_RGB8_texture, GL_OES_draw_elements_base_vertex, GL_OES_standard_derivatives, GL_ANGLE_pack_reverse_row_order, GL_EXT_instanced_arrays, GL_OES_EGL_image_external, GL_EXT_texture_compression_dxt1, GL_OES_depth32, GL_OES_get_program_binary, GL_EXT_map_buffer_range, GL_ANGLE_get_tex_level_parameter, GL_EXT_color_buffer_float, GL_EXT_texture_compression_s3tc_srgb, GL_NV_pixel_buffer_object, GL_ANGLE_base_vertex_base_instance, GL_ANGLE_request_extension, GL_EXT_EGL_image_external_wrap_modes, GL_OES_texture_float, GL_CHROMIUM_bind_generates_resource, GL_OES_packed_depth_stencil, GL_ANGLE_texture_compression_dxt3, GL_OVR_multiview2, GL_EXT_blend_func_extended, GL_ANGLE_multi_draw, GL_EXT_texture_format_BGRA8888, GL_EXT_draw_elements_base_vertex, GL_EXT_debug_marker, GL_CHROMIUM_copy_compressed_texture, GL_EXT_read_format_bgra, GL_EXT_texture_compression_bptc, GL_EXT_texture_rg, GL_OES_depth24, GL_OES_element_index_uint, GL_OES_compressed_ETC2_RGBA8_texture, GL_EXT_shader_texture_lod, GL_EXT_texture_storage, GL_KHR_parallel_shader_compile, GL_NV_fence, GL_EXT_texture_filter_anisotropic, GL_ANGLE_translated_shader_source, GL_EXT_robustness, GL_ANGLE_robust_client_memory, GL_CHROMIUM_sync_query, GL_ANGLE_client_arrays, GL_CHROMIUM_bind_uniform_location, GL_EXT_sRGB
[2023.07.24 12:32:28] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2023.07.24 12:32:28] OpenGL: QOpenGLContext created, version: 3.3.
[2023.07.24 12:32:28] OpenGL: [TRUE] (Window)
[2023.07.24 12:32:28] Update Info: MTP is unavailable.
[2023.07.24 12:33:02] Export Info: Destroy top bar by controller removal.
[2023.07.24 12:33:03] App Info: writing encrypted user settings...
[2023.07.24 12:33:03] MTP Info: -404 error received in 160203 with temporary key, assuming it was destroyed.
[2023.07.24 12:33:04] Message Info: bad message notification received (error_code 33) for msg_id = 7259315863799412744, seq_no = 22
[2023.07.24 12:33:04] Message Info: bad message notification received, msgId 7259315863799412744, error_code 33
[2023.07.24 12:33:04] Message Info: bad message notification received (error_code 33) for msg_id = 7259315863962201580, seq_no = 48
[2023.07.24 12:33:04] Message Info: bad message notification received, msgId 7259315863962201580, error_code 33
[2023.07.24 12:33:05] Assertion Failed! "value.bytes.size() <= _settings.maxDataSize" storage_cache_database_object.cpp:807
```

**Comment by Guzya** [comment]
+1 (Linux Mint 21.1 Cinnamon)
 #26570

**Comment by All-The-Foxes** [comment]
Up until this last update 4.8.8 just now, mine crashed whenever I opened it anywhere from a few seconds in to just around a minute after opening the client. I could leave it open and tab out and it would crash, immediately start a call and have it crash, nothing I did or didn't do made a difference. Enabling beta versions and letting it crash didn't produce a crash log, but if I opened it through terminal it wouldn't crash. It seems to be fixed now though.
